### PR TITLE
Multi Chat, DiscordService and npm run test

### LINKED ISSUE #1: Unable to Connect to Discord
I tried using this to connect to discord and it didn't work

### PR DESCRIPTION
**Pull request by rkrasnan**
Created chats/ to store multiple chat classes. 

Moved slack logic into SlackService class. 
Created DiscordService class. 

Implemented $ npm run test and /test/index.ts to enable running locally

### CODE REVIEW

**Comment by rkrasnan** [comment]
FOODSBY_CHANNEL is now SLACK_CHANNEL

File: package.json [hunk 1]
```diff
@@ -5,16 +5,19 @@
   "main": "index.js",
   "scripts": {
     "build": "rm -rf dist && webpack",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "rm -rf dist && tsc && node dist/test/index.js"
```
Comment (tzfx) [inline package.json:8]: You can actually refer to other npm scripts inside of other npm scripts, so: `npm run build && node dist/test/index.js` here.

File: package.json [hunk 2]
```diff
@@ -5,16 +5,19 @@
...
   },
   "author": "adamv.stadler@gmail.com",
   "license": "ISC",
   "dependencies": {
     "@slack/web-api": "^5.13.0",
+    "discord.js": "^12.5.0",
+    "ffmpeg-static": "^4.2.7",
```
Comment (tzfx) [inline package.json:15]: What's ffmpeg for? Co-dependency for discord or something of the such?

**Comment by tzfx** [comment]
Closes Issue #1

File: package.json (hunk 2 above)
Comment (rkrasnan) [reply package.json:15]: a discord dependency that wasn't properly included in the discord package

**Pull request merged**